template/README.md.jinja

```diff
@@ -12,7 +12,7 @@ how it does it, and why people should use it.
 
 Source          | <{{repo_url}}>
 :---:           | :---:
-PyPI            | `pip install {{distribution_name}}`
+{% if pypi %}PyPI            | `pip install {{distribution_name}}`{% endif %}
```

Comment:
Just thought, if pypi and sphinx are not specified, does this give a couple of blank lines in the table? Please can you try making one of these and see how it renders on GitHub?

Reply:
yes I tried this before and it does leave line gaps